src/registrar/templates/emails/domain_invitation.txt

```diff
@@ -1,6 +1,25 @@
-You have been invited to manage the domain {{ domain.name }} on get.gov, 
-the registrar for .gov domain names.
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
```

Comment:
Is {% autoescape off %} something we need to implement on all of our txt files?

Reply:
I'm not sure, but just following the pattern of other txt files!